cmd/client/main.go

```diff
@@ -113,8 +113,8 @@ func main() {
 		completionCmd,
 	)
 
-	rootCmd.PersistentFlags().StringVar(&command.CommandlineGlobalFlags.Server,
-		"server", "localhost:2381", "The address of the Easegress endpoint")
+	rootCmd.PersistentFlags().StringVar(&command.CommandlineGlobalFlags.URL,
+		"url", "https://localhost:2381", "The address of the Easegress endpoint")
```

Comment:
Should the help message be `http://localhost:2381` since the admin portal is HTTP by default?